lib/npg_pipeline/product/release/irods.pm

```diff
@@ -84,6 +87,28 @@ sub is_for_irods_release {
   return $enable;
 }
 
+sub _siblings_are_for_irods_release {
+  my ($self, $product) = @_;
+
+  $product->lims or croak 'Need lims object';
+
+  my @lims = ();
+  foreach my $p ($product->lanes_as_products()) {
+    my $c = $p->composition->get_component(0);
+    my $l = $product->lims->create_lane_object($c->id_run, $c->position);
```

Comment:
Guessing that we cannot _just_ do `my$l = $p->lims` as `lanes_as_products()` does not populate/provide it?
(Style question - should this `lims` be lazily built in the object returned from `lanes_as_products()` - not possible? - not worth it now?)

Reply:
This is now done